RFC-004-Bitcoin.md

```diff
@@ -0,0 +1,54 @@
+# Bitcoin Definitions
+
+As per [Bitcoin Core](https://github.com/bitcoin/bitcoin/) concensus.
+
+Introduced for RFC-003 (should RFC-003 be part of the title?)
```

Comment:
The introduction _definitely_ needs to be much more verbose. I think it should contain a 2-3 sentence paragraph explaining what this RFC is about.

Reply:
Yes but is this RFC about RFC-003? That is the question.